x/ibcratelimit/module/ibc_middleware_test.go

```diff
@@ -124,86 +126,58 @@ func (s *MiddlewareTestSuite) MessageFromBToA(denom string, amount sdkmath.Int)
 	)
 }
 
-// Tests that a receiver address longer than 4096 is not accepted
-func (s *MiddlewareTestSuite) TestInvalidReceiver() {
-	msg := transfertypes.NewMsgTransfer(
-		s.path.EndpointB.ChannelConfig.PortID,
-		s.path.EndpointB.ChannelID,
-		sdk.NewInt64Coin(sdk.DefaultBondDenom, 1),
-		s.chainB.SenderAccount.GetAddress().String(),
-		strings.Repeat("x", 4097),
-		clienttypes.NewHeight(0, 100),
-		0,
-		"",
-	)
-	_, ack, _ := s.FullSendBToA(msg)
-	s.Assert().Contains(ack, "error",
-		"acknowledgment is not an error")
-	s.Assert().Contains(ack, fmt.Sprintf("ABCI code: %d", ibcratelimit.ErrBadMessage.ABCICode()),
-		"acknowledgment error is not of the right type")
-}
-
 // FullSendBToA does the entire logic from sending a message from chain B to chain A.
-func (s *MiddlewareTestSuite) FullSendBToA(msg sdk.Msg) (*sdk.Result, string, error) {
-	sendResult, err := s.chainB.SendMsgsNoCheck(msg)
-	s.Assert().NoError(err)
-
-	packet, err := ibctesting.ParsePacketFromEvents(sendResult.Events)
-	s.Assert().NoError(err)
-
-	err = s.path.EndpointA.UpdateClient()
-	s.Assert().NoError(err)
+func (suite *MiddlewareTestSuite) FullSendBToA(msg sdk.Msg) (*abci.ExecTxResult, string, error) {
+	sendResult, err := suite.chainB.SendMsgsNoCheck(&suite.Suite, msg)
+	suite.Require().NoError(err)
 
-	res, err := s.path.EndpointA.RecvPacketWithResult(packet)
-	s.Assert().NoError(err)
+	packet, err := ibctesting.ParsePacketFromEvents(sendResult.GetEvents())
+	suite.Require().NoError(err)
 
+	err = suite.path.EndpointA.UpdateClient()
+	suite.Require().NoError(err)
+	res, err := suite.path.EndpointA.RecvPacketWithResult(packet)
+	suite.Require().NoError(err)
 	ack, err := ibctesting.ParseAckFromEvents(res.GetEvents())
-	s.Assert().NoError(err)
-
-	err = s.path.EndpointA.UpdateClient()
-	s.Assert().NoError(err)
-	err = s.path.EndpointB.UpdateClient()
-	s.Assert().NoError(err)
-
+	suite.Require().NoError(err)
+	err = suite.path.EndpointA.UpdateClient()
+	suite.Require().NoError(err)
+	err = suite.path.EndpointB.UpdateClient()
+	suite.Require().NoError(err)
 	return sendResult, string(ack), err
 }
 
 // FullSendAToB does the entire logic from sending a message from chain A to chain B.
-func (s *MiddlewareTestSuite) FullSendAToB(msg sdk.Msg) (*sdk.Result, string, error) {
-	sendResult, err := s.chainA.SendMsgsNoCheck(msg)
+func (suite *MiddlewareTestSuite) FullSendAToB(msg sdk.Msg) (*abci.ExecTxResult, string, error) {
```

Comment:
I really dislike using `suite` for these receivers instead of `s`. For one thing, it overshadows the imported package name. And even though it's only an extra 4 characters, it feels like it makes lines a lot longer when fully spelled out.

Reply:
I actually prefer to keep it this way to match the source. I'll actually update the others to match the source as well.